src/ansys/templates/python/osl_solution/{{cookiecutter.__project_name_slug}}/src/ansys/solutions/{{cookiecutter.__solution_name_slug}}/solution/problem_setup_step.py

```diff
@@ -333,8 +333,10 @@ def start_and_monitor_osl_project(self, osl_manager: OptislangManager) -> None:
                     for hid in actor_states_ids:
                         # Get actor status info (TCP REQUEST)
                         actor_status_info = self._make_osl_server_request(osl, "get_actor_status_info", actor_uid=node_props["uid"], hid=hid)
+                        # Get actor info (TCP REQUEST)
+                        actor_info = self._make_osl_server_request(osl, "get_actor_info", actor_uid=node_props["uid"], hid=hid)
```

Comment:
@shuklaprathmesh can we remove this statement as the get_actor_info query is already made on line 322 and can be reused here. Thanks.